server/odc-migrate/src/main/resources/migrate/common/V_4_2_2_4__add_dlm_task_meta.sql

```diff
@@ -0,0 +1,31 @@
+CREATE TABLE IF NOT EXISTS `dlm_task_generator` (
+`id` bigint(20) NOT NULL AUTO_INCREMENT,
+`create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT '创建时间',
+`update_time` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '修改时间',
+`generator_id` varchar(120) NOT NULL,
+`job_id` varchar(120) NOT NULL,
+`processed_data_size` bigint(20) NOT NULL,
+`processed_row_count` bigint(20) NOT NULL,
+`status` varchar(64) NOT NULL,
+`type` varchar(32) NOT NULL,
+`primary_key_save_point` varchar(512) DEFAULT NULL,
+`partition_save_point` varchar(512) DEFAULT NULL,
+`task_count` bigint(20) NOT NULL DEFAULT '0',
+PRIMARY KEY (`id`),
```

Comment:
do not use anonymous pk, use `constraint pk_ dlm_task_generator_id primary key(`id`)` instead